proxy/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo '* * * * * node /cfbs-proxy/processStats.js >> /home/proxy/cron.log 2>&1' > /etc/crontabs/nginx
```

Comment:
This can fail in a horrible way if the script processStats.js doesn't complete within one minute, which can be the case if the log file is huge. Please consider the usage of flock to create a file-based semaphore avoiding multiple parallel runs.

Reply:
Yeah, it looks to me like this should just be running forever and have the sleep implemented in the code. *inotify* or using systemd *.path* unit would be much better for this, though.

Reply:
OK, I will rewrite script to be running forever with a sleep and https://github.com/foreversd/forever to ensure that script is running